### PR TITLE
CLDR-17423 Placeholder for MessageFormat 2.0 spec, part 9

### PR DESCRIPTION
**Pull request by srl295**
- final spec will be poured in as CLDR-17424

CLDR-17423

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by srl295** [comment]
fyi @aphillips @eemeli

File: docs/ldml/tr35-collation.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ The LDML specification is divided into the following parts:
 *   Part 6: [Supplemental](tr35-info.md#Contents) (supplemental data)
 *   Part 7: [Keyboards](tr35-keyboards.md#Contents) (keyboard mappings)
 *   Part 8: [Person Names](tr35-personNames.md#Contents) (person names)
+*   Part 9: [Message Format](tr35-messageFormat.md#Contents) (message format)
```
Comment (aphillips) [inline docs/ldml/tr35-collation.md:47]: Generally we refer to it as MessageFormat without a space

```suggestion
Part 9: [MessageFormat](tr35-messageFormat.md#Contents) (message format)
```

File: docs/ldml/tr35-messageFormat.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+## Unicode Technical Standard #35
+
+# Unicode Locale Data Markup Language (LDML)<br/>Part 9: Message Format
+
+|Version|45 (draft)              |
+|-------|------------------------|
+|Editors|Addison Phillips and [other CLDR committee members](tr35.md#Acknowledgments)|
```
Comment (aphillips) [inline docs/ldml/tr35-messageFormat.md:7]: Do we need to develop a separate acknowledgements section?

Also, how many names can we list as editors? A core group of contributors might be nice to have here.

File: docs/ldml/tr35-messageFormat.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+Publication does not imply endorsement by the Unicode Consortium.
+This is not a stable document; it is inappropriate to cite this document as other than a work in progress._
+
+<!-- _This document has been reviewed by Unicode members and other interested parties, and has been approved for publication by the Unicode Consortium.
+This is a stable document and may be used as reference material or cited as a normative reference by other specifications._ -->
+
+> _**A Unicode Technical Standard (UTS)** is an independent specification. Conformance to the Unicode Standard does not imply conformance to any UTS._
+
+_Please submit corrigenda and other comments with the CLDR bug reporting form [[Bugs](tr35.md#Bugs)]. Related information that is useful in understanding this document is found in the [References](tr35.md#References). For the latest version of the Unicode Standard see [[Unicode](tr35.md#Unicode)]. For a list of current Unicode Technical Reports see [[Reports](tr35.md#Reports)]. For more information about versions of the Unicode Standard, see [[Versions](tr35.md#Versions)]._
```
Comment (aphillips) [inline docs/ldml/tr35-messageFormat.md:28]: Can we change this bit of boilerplate? We'll want to use a difference mechanism.

**Review by aphillips — COMMENTED** [review]
Thanks for this. Some questions follow.

File: docs/ldml/tr35-collation.md (hunk 1 above)
Comment (srl295) [reply docs/ldml/tr35-collation.md:47]:
```suggestion
*   Part 9: [MessageFormat](tr35-messageFormat.md#Contents) (message format)
```

File: docs/ldml/tr35-messageFormat.md (hunk 2 above)
Comment (srl295) [reply docs/ldml/tr35-messageFormat.md:7]: could be a future update - past Monday's deadline

File: docs/ldml/tr35-messageFormat.md (hunk 3 above)
Comment (srl295) [reply docs/ldml/tr35-messageFormat.md:28]: could be a future update - past Monday's deadline

File: docs/ldml/tr35-messageFormat.md (hunk 2 above)
Comment (aphillips) [reply docs/ldml/tr35-messageFormat.md:7]: Fair enough for both of these questions. But I'd still like to know the answer (might not be you who can answer it). I can update both of these by Monday.

**Comment by macchiati** [comment]
We have two acknowledgements lists.
In the spec: https://unicode.org/reports/tr35/#Acknowledgments
The big list: https://cldr.unicode.org/index/acknowledgments

We can add people to the spec or to both. The spec one tends to be major,
and also can list what they contributed. If you feel like those choices are
not optimal, we can discuss other options in the TC.

File: docs/ldml/tr35-collation.md (hunk 1 above)
Comment (srl295) [reply docs/ldml/tr35-collation.md]: done

**Comment by srl295** [comment]
@aphillips we also might consider putting this in, so that there's a part 9 to start pointing people to come Monday.. we'll already have a followon PR to put the actual text in, can easily make other updates.

**Comment by aphillips** [comment]
This PR is fine by me. I was hoping to get my paragraph out of message-format-wg#697, but we can change whatever we need to after Monday, I presume.

File: docs/ldml/tr35-messageFormat.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+|-------|------------------------|
+|Editors|Addison Phillips and [other CLDR committee members](tr35.md#Acknowledgments)|
+
+For the full header, summary, and status, see [Part 1: Core](tr35.md).
+
+### _Summary_
+
+This specification defines the data model, syntax, processing, and conformance requirements for the next generation of dynamic messages.
+
+This is a partial document, describing only those parts of the LDML that are relevant for message format. For the other parts of the LDML see the [main LDML document](tr35.md) and the links above.
```
Comment (aphillips) [inline docs/ldml/tr35-messageFormat.md:15]: This sentence doesn't make sense? There are no parts of LDML here, just the MF2 spec?

```suggestion
For the other parts of LDML see [main LDML document](tr35.md) and the links above.
```

**Review by aphillips — APPROVED** [review]
Approving so we have something on Monday, 4 March.

File: docs/ldml/tr35-messageFormat.md (hunk 4 above)
Comment (srl295) [reply docs/ldml/tr35-messageFormat.md:15]: Yes, there are other parts of LDML.  This is part 9 of 9 parts.

Comment (srl295) [reply docs/ldml/tr35-messageFormat.md:15]: all other parts have the same wording.

**Review by macchiati — COMMENTED** [review]
approved (with Addison's agreement) as per email discussion.

**Review by macchiati — APPROVED** [review]

**Pull request merged**